oem/ibm/libpldmresponder/oem_ibm_handler.hpp

```diff
@@ -65,11 +66,6 @@ namespace oem_ibm_platform
 constexpr uint16_t ENTITY_INSTANCE_0 = 0;
 constexpr uint16_t ENTITY_INSTANCE_1 = 1;
 
-constexpr uint32_t BMC_PDR_START_RANGE = 0x00000000;
```

Comment:
I guess its better to have these constants in oem/ibm rather than common/utils as this is IBM specific ranges